src/init/CliRunner.ts

```diff
@@ -44,6 +44,10 @@ export class CliRunner {
             if (!args[key]) {
               throw new Error(`Missing value for argument "${key}"`);
             }
+            // Check if the argument only has 1 value
+            if (Array.isArray(args[key])) {
+              throw new Error(`Multiple values were provided for: "${key}", [${args[key]}]`);
+            }
```

Comment:
Doesn't

```suggestion
            if (typof args[key] !== string') {
              throw new Error(`Missing value for argument "${key}"`);
            }
```

work?

Reply:
Never mind, there might be booleans etc.